ZupIT/ritchie-cli
# [FEATURE] cmd init

**- What I did**
Add a new command`init` that will run a initial setup for `rit`.
 - Single: Requires the passphrase
 - Team: Requires the name and serverURL of the organization. It will possible to peform login too (optional).

**- How I did it**
 - Add a new cobra command `init` supporting prompt inputs and stdin
 - Change PreRun func in cobra command `root`
 - Change `serverURL string` to `server.Config struct` with fields _(Organization, URL)_
 - Remove unecessary files

**- How to verify it**
 - clean the rit home
 - run any command, check the output message
 - run `rit init` and perform the setup 

**- Description for the changelog**

A new command`init` that will run a initial setup for `rit`.
